local/client/conn/conn.go

```diff
@@ -0,0 +1,66 @@
+package conn
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"time"
+)
+
+// A Dialer can dial a connection to the Scoot server.
```

Comment:
I agree. "//" by default, no big deal if someone forgets and uses "/**/". I imagine that godoc, if we use that, supports either.